lib/maintainers/maintainer-list.nix

```diff
@@ -62,4 +62,8 @@ See `<nixpkgs/maintainers/scripts/check-maintainer-github-handles.sh>` for an ex
     email = "stephane.tzvetkov@cea.fr";
     name = "Stéphane Tzvetkov";
   };
+  agaget = {
```

Comment:
Can you put yourself at the top? The comment above says to keep the list alphabetically sorted

Reply:
I'm always at the top 👓